rpm/registryquery.spec

```diff
@@ -26,7 +26,8 @@ Requires:       python-requests           >= 0.9.3
 Requires:       python-ws4py
 Requires:       ips-etcd
 Requires:       ips-reverseproxy-common
-Requires:	nmoscommon
+Requires:       nmoscommon
+Requires:       python-nmos-oauth
```

Comment:
As with the other PR(s) I suspect the nmos-oauth dependencies can be removed now